plugins/jira/index.js

```diff
@@ -84,30 +85,31 @@ class JiraPlugin extends BasePlugin {
   async findTicketsAndPost(commenter, ticketIds) {
     const jql = `id in ('${ticketIds.join("', '")}')`;
 
-    const res = await request.post(`${this.jiraConfig.protocol}://${this.jiraConfig.host}/rest/api/2/search`, {
-      json: true,
-      resolveWithFullResponse: true,
+    const res = await fetch(`${this.jiraConfig.protocol}://${this.jiraConfig.host}/rest/api/2/search`, {
+      method: 'POST',
       headers: {
-        Accept: 'application/json'
+        'Accept': 'application/json',
+        'Authorization': 'Basic ' + Buffer.from(this.jiraConfig.username + ':' + this.jiraConfig.password)
+          .toString('base64'),
+        'Content-Type': 'application/json'
       },
-      auth: {
-        username: this.jiraConfig.username,
-        password: this.jiraConfig.password
-      },
-      body: {
+      body: JSON.stringify({
         jql,
         startAt: 0,
         fields: ['summary']
-      }
+      })
     });
 
-    if (!res || res.statusCode < 200 || res.statusCode > 299)
+    if (!res || res.status < 200 || res.status > 299) {
```

Comment:
you could change this to 
```suggestion
    if (!res || !res.ok) {
```